src/strategies/MangroveOrder.sol

```diff
@@ -204,19 +209,30 @@ contract MangroveOrder is Forwarder, IOrderLogic {
     // POST (else)
     // * (NAT_USER+`res.bounty`, OUT_USER+`res.takerGot`, IN_USER-`res.takerGave`)
     // * (NAT_THIS, OUT_THIS, IN_THIS)
-    emit OrderSummary({
-      mangrove: MGV,
-      outbound_tkn: tko.outbound_tkn,
-      inbound_tkn: tko.inbound_tkn,
-      fillWants: tko.fillWants,
-      taker: msg.sender,
-      takerGot: res.takerGot,
-      takerGave: res.takerGave,
-      penalty: res.bounty
-    });
+    logOrderData(tko, res);
     return res;
   }
 
+  function logOrderData(TakerOrder memory tko, TakerOrderResult memory res) internal {
+    emit OrderSummary(
+      MGV,
+      tko.outbound_tkn,
+      tko.inbound_tkn,
+      msg.sender,
+      tko.fillOrKill,
+      tko.takerWants,
+      tko.takerGives,
+      tko.fillWants,
+      tko.restingOrder,
+      tko.expiryDate,
```

Comment:
the expiry date is emitted in expiry events, right? So it seems redundant.

Reply:
This PR is following the self-contained approach as discussed in the PR description.